ui/apps/platform/src/Containers/Integrations/IntegrationForm/Forms/S3IntegrationForm.tsx

```diff
@@ -28,6 +28,7 @@ export type S3Integration = {
         useIam: boolean;
         accessKeyId: string;
         secretAccessKey: string;
+        usePathstyle: boolean;
```

Comment:
You added this to the type, but you didn't add a default value to the `defaultValues` seed object for new integrations, down around line 100 and following.

JS is very forgiving, so the absence of a value will automatically coerce to `false` but better to be explicit.